MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/handlers/setup/items/category/MaterialMultiLocationItem.java

```diff
@@ -249,7 +264,7 @@ private void removeLocation(HumanEntity player, boolean deleteAll) {
           setupInventory.setConfig(keyName, null);
           //considerable to add arena method to remove location
           new MessageBuilder("&e✔ Removed | &a" + name.toUpperCase() + " location for arena " + setupInventory.getArenaKey() + "! (" + location + ")").prefix().send(player);
-          new MessageBuilder("You can now remove the " + checkMaterial.toString() + "!").prefix().send(player);
+          new MessageBuilder("You can now remove the " + checkMaterials.toString() + "!").prefix().send(player);
```

Comment:
could return unreadable long message for user, getting the "clicked" type would be more accurate